File: Python/Find the Difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        ls = list(s)
        lt = list(t)
        ls.sort()
        lt.sort()
        ls.append(None)
        for i , j in zip(ls, lt):
            if i != j:
                return j

File: cpp/Find the Difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        char res = 0;
        for (char ch : s) {
            res ^= ch;
        }
        for (char ch : t) {
            res ^= ch;
        }
        return res;
    }
};
